src/services/NetworkService/realtimeUpdates.ts

```diff
@@ -1,33 +1,59 @@
 import { constants } from '../AppService';
 import { io, Socket } from 'socket.io-client';
+import { SdkManager } from '../common';
 
 export class RealtimeUpdates {
-  private socket: Socket;
+  private socket!: Socket;
 
-  constructor() {
-    const URL = `${constants.DRIVE_API_URL}`;
-    this.socket = io(URL, {
+  constructor(private sdk: SdkManager) {}
+
+  init() {
+    this.socket = io(constants.NOTIFICATIONS_URL, {
+      auth: {
+        token: this.sdk.getApiSecurity().photosToken,
```

Comment:
the naming is a bit confusing, is not a `photosToken`, is the new auth token with the proper format.

Reply:
I know, but the name was already this when I started on mobile, is on the list of pending refactors, is maintained for compatibility and consistency, but will be definitely refactorized in the near future